scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
+    if unlisted_addons is None:
+        unlisted_addons = UNLISTED_ADDONS
+    return (result
+            for result in results
+            if (not result['metadata'].get('listed', True)
+                or result['metadata'].get('id') in unlisted_addons))
```

Comment:
I just did a fresh fetch from the db for the range 2015-06-28 through 2015-07-27 (previous results were 2015-06-21 through 2015-07-20).

```
((None, False, True), 878)
((False, False, False), 223)
((False, False, True), 522)
((False, True, False), 23)
((True, False, True), 58)